src/app.ts

```diff
@@ -46,6 +60,27 @@ app.use(validator({
   schemaPath: path.join(__dirname, '../node_modules/@badgateway/tt-types/schema')
 }));
 
+// a12n setup
+const client = new OAuth2Client({
+  server: 'http://localhost:8531',
+  clientId: 'tt-api',
+  clientSecret: process.env.OAUTH2_CLIENT_SECRET,
+  tokenEndpoint: '/token',
+  introspectionEndpoint: '/introspect',
+});
+
+app.use(browserToBearer({client}));
+
+app.use(oauth2({
+  publicPrefixes: [
+    '/health-check',
+    '/login',
+    '/register'
```

Comment:
Where do these come from?

Reply:
not sure, I was doing a lot of copying and pasta'ing from `media-api`

i see there were a lot of things I missed! another commit coming soon!